src/cast_core/src/CrudActor.cpp

```diff
@@ -810,6 +810,40 @@ struct FindOperation : public BaseOperation {
     metrics::Operation _operation;
 };
 
+struct FindOneOperation : public BaseOperation {
```

Comment:
Changes in this file are part of a separate PR that hasn't been merged yet https://github.com/mongodb/genny/pull/364